src/slidedown/Slidedown.ts

```diff
@@ -143,14 +143,18 @@ export default class Slidedown {
 
     if (state) {
       // note: savingButtonText is hardcoded in constructor. TODO: pull from config & set defaults for future release
-      this.allowButton.innerHTML = this.getIndicatorHolderHtmlWithText(this.categoryOptions.savingButtonText).innerHTML;
       this.allowButton.disabled = true;
+      this.allowButton.textContent = null;
```

Comment:
Is it necessary to remove the `textContent`?

Reply:
Yeah, otherwise it will just get appended to the end and we'll have the previous text plus the new text and the spinning indicator

Reply:
👍 ah gotcha